resources/views/themes/default1/order/index.blade.php

```diff
@@ -99,53 +99,28 @@
                 {!! Form::select('act_inst',[null => 'Select']+ $activeInstallationOptions, $request->act_inst, ['class' => 'form-control','id'=>'act_inst']) !!}
             </div>
 
-             <div class="col-md-3 form-group">
-                 {!! Form::label('version_less_than_equal','Version less than or equal to') !!}
-                 {!! Form::select('version_less_than_equal',[null => 'Select']+ array_combine($allVersions, $allVersions), $request->version_less_than_equal,
-                ['class' => 'form-control','id'=>'version_less_than_equal']) !!}
-
-             </div>
-
             <div class="col-md-3 form-group">
-                {!! Form::label('version_greater_than_equal','Version greater than or equal to') !!}
-                {!! Form::select('version_greater_than_equal',[null => 'Select']+ array_combine($allVersions, $allVersions), $request->version_greater_than_equal,
-                ['class' => 'form-control','id'=>'version_greater_than_equal']) !!}
+                {!! Form::label('version_from','Version From') !!}
+                {!! Form::select('version_from',[null => 'Select']+ array_combine($allVersions, $allVersions), $request->version_from,
```

Comment:
We are using 'Choose' instead of 'Select' in the system.

Reply:
There was an issue if you remember to disable values coming from cache in THE date range fields. If we have selected dates before for searching, it shows recommedations to select those dates. This sometimes covers the date picker. Can this be also fixed in this PR?

Reply:
> We are using 'Choose' instead of 'Select' in the system.

fixed

Reply:
> There was an issue if you remember to disable values coming from cache in THE date range fields. If we have selected dates before for searching, it shows recommedations to select those dates. This sometimes overlaps with the date picker. Can this be also fixed in this PR?

sure. fixed